tests/Get-FilteredRestoreFile.Tests.ps1

```diff
@@ -4,7 +4,7 @@ Describe "Get-FilteredRestoreFile Unit Tests" -Tag 'Unittests'{
     Context "Empty TLog Backup Issues" {
         $Header = ConvertFrom-Json -InputObject (Get-Content $PSScriptRoot\..\tests\ObjectDefinitions\BackupRestore\RawInput\EmptyTlogData.json -raw)
         Mock Read-DbaBackupHeader { $Header }
-        $Output = Get-FilteredRestoreFile -SqlServer 'TestSQL' -Files "c:\dummy.txt"
+        $Output = Get-FilteredRestoreFile -SqlServer TestSQL -Files "c:\dummy.txt" -silent $true
```

Comment:
Valid since you want to force it but $Silent should be a switch and never a bool so this should be -Silent:$silent eh?